lib/conrad/collector.rb

```diff
@@ -156,13 +179,23 @@ def record_events_as_batch
 
     def record_individual_events
       events.each do |event|
-        emitter.call(event)
+        begin
+          emitter.call(event)
+        rescue StandardError => e
```

Comment:
Nit: the rescue default is standard error, so you can `rescue => e`

Reply:
Rubocop prefers you rescue *something*, even if that means duplicating the default.

Reply:
Fair enough. You win this round, rubocop